src/sequelize/seeders/5_user_seed.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  up: (queryInterface) => queryInterface.bulkInsert('Users', [{
+    firstName: 'foo',
+    lastName: 'bar',
+    userName: 'foobar',
+    email: 'foobar@gmail.com',
+    password: 'sdfssgggg',
```

Comment:
I think the password needs to be hashed, and there are some missing fields like phone number and gender. please check the other sheet with all table to see all required fields.

Reply:
For some reason they are not required. But I just wanted the userId. You can create a seed that has all the data you need